File: Final 2018-3/srcgen/Direccion.cpp
class Direccion{

  Direccion(this){
  }
  Direccion((this,calle,ciudad,pais):
      this.calle=calle
      this.ciudad=ciudad
      this.pais=pais

  void getCalle(){
    return this.calle
	}
  void getCiudad(){
    return this.ciudad
	}
  void getPais(){
    return this.pais
	}
  String setCalle(this,calle){
    this.calle = calle
  }

  String setCiudad(this,ciudad){
    this.ciudad = ciudad
  }

  String setPais(this,pais){
    this.pais = pais
  }

};

File: Final 2018-3/srcgen/Direccion.rb
	
class Direccion

	def initialize(this,calle,ciudad,pais)
		@calle = calle
		@ciudad = ciudad
		@pais = pais
	end
	
	def calle
		return @calle
	end

	def ciudad
		return @ciudad
	end

	def pais
		return @pais
	end

    def calle=(calle)
		@calle = calle
    end
	
    def ciudad=(ciudad)
		@ciudad = ciudad
    end
	
    def pais=(pais)
		@pais = pais
    end
	
end